src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -163,7 +163,7 @@ public void unsetRotationTarget() {
         @Override
         public void periodic() {
                 if (chassisSpeeds.omegaRadiansPerSecond == 0 && (chassisSpeeds.vxMetersPerSecond != 0 || chassisSpeeds.vyMetersPerSecond != 0)) {
-                        chassisSpeeds.omegaRadiansPerSecond = -rotationPID.calculate(getGyroscopeRotation().minus(rotationTarget).getRadians());
+                        chassisSpeeds.omegaRadiansPerSecond = rotationPID.calculate(getGyroscopeRotation().minus(rotationTarget).getRadians());
```

Comment:
Did the front of the robot flip? I would have expected the sign to change here

Reply:
Without sign change there was oscillation, with sign change behavior was as expected.